Dockerfile

```diff
@@ -13,14 +13,10 @@ ENV DB.USER=""
 ENV DB.PASSWORD=""
 ENV DB.CHOICE="SQLLITE"
 
-RUN mkdir -p /usr/src/app
```

Comment:
Wouldnt doing this make the dockerfile a dev oriented dockerfile?
If not, we would need to have a way to deploy the code in PROD.
One possible way could be to stick a setup.py which would actually build the code as an installable and that gets stuck in the Dockerfile..